.circleci/config.yml

```diff
@@ -790,6 +805,24 @@ jobs:
       - upload-artifacts-snapshot
       - trigger-mobile-metrics
 
+  release-weekly-snapshot:
+    executor: ndk-r22-latest-executor
+    resource_class: medium+
+    # Allow snapshot repository for snapshot releases
+    environment:
+      ALLOW_SNAPSHOT_REPOSITORY: true
+      WEEKLY_SNAPSHOT_WORKFLOW: true
+    steps:
+      - checkout
+      - checkout-branch
+      - assemble-core-release
+      - assemble-ui-release
+      - prepare-mbx-ci
+      - setup-aws-credentials
+      - upload-artifacts-snapshot
+      - trigger-mobile-metrics
+      - trigger-onetap
```

Comment:
When we do a release as usual, it is not available immediately and we need to wait for about half an hour before it can be integrated into 1TAP. Won't we run into the same issue here?

Reply:
No, it's just a snapshot in s3. I've tested